packages/components/src/components/header/header.component.ts

```diff
@@ -0,0 +1,141 @@
+/* eslint-disable @typescript-eslint/unbound-method */
+import { classMap } from 'lit/directives/class-map.js';
+import { html } from 'lit/static-html.js';
+import { property } from 'lit/decorators.js';
+import type { CSSResultGroup } from 'lit';
+import SynergyElement from '../../internal/synergy-element.js';
+import { HasSlotController } from '../../internal/slot.js';
+import SynIconButton from '../icon-button/icon-button.component.js';
+import componentStyles from '../../styles/component.styles.js';
+import styles from './header.styles.js';
+
+/**
+ * @summary The <syn-header /> element provides a generic application header
+ * that can be used to add applications name, toolbar and primary navigation.
+ *
+ * @documentation https://synergy-design-system.github.io/?path=/docs/components-syn-header--docs
+ * @status stable
+ * @since 1.8.0
+ *
+ * @dependency syn-icon-button
+ *
+ * @slot - The label for the header. Will automatically be hidden on mobile.
+ * @slot logo - The logo that should be displayed. Will fall back to the SICK logo if not applied.
+ * @slot option-menu - Used to add various application toolbar icons.
+ *                     Best used with `<syn-icon-button />` and `<syn-drop-down />`
+ * @slot top-navigation - Used to add an optional horizontal navigation
+ *
+ * @csspart base - The component's base wrapper.
+ * @csspart side-navigation-button - Can be used to apply styles to the side-navigation menu icon
```

Comment:
From my feeling we should skip this part completely in this PR. I would wait with anything like styling or whatever, until we build the side-nav as well. What do you think? From my feeling this lays too much foundation for things that may change when building the side-nav. (And I thought that we explicitly excluded everything related to that in the ticket – I just saw, that it jsut handles "interaractivity", but from my perspective it should skip the side-navigation part completely ATM.

Reply:
I would propose to copy the parts for the side nav into the other ticket, so that this clearly stated as not shipping it.

Reply:
My expectation was that it just handles the foundation for the horizontal-navigation (which is not connected for an interactivity perspective) and the top bar things.

Reply:
> My expectation was that it just handles the foundation for the horizontal-navigation (which is not connected for an interactivity perspective) and the top bar things.

This is now true again. I initially added the menu item as it has its own layout requirements, but I also agree it is saver to not ship it yet.